### PR TITLE
Feat/textarea

### PR DESCRIPTION
**Pull request by levanion**
#### Basic
- [ ] Used plop (`npm run plop`) to create a new component.
- [ ] PR has description.
- [ ] New component is functional and uses Hooks. 
- [ ] Component defines [`PropTypes`](https://reactjs.org/docs/typechecking-with-proptypes.html).
#### Style
- [ ] Styles are added to `NewComponent.modules.scss` file inside of the `NewComponent` folder.
- [ ] Component uses CSS Modules.
- [ ] Design is compatible with [L3vels Design System](https://design.l3vels.xyz/).
#### Storybook
- [ ] Stories were added to `/src/NewComponent/__stories__/NewComponent.stories.js` file.
- [ ] Stories include all flows of using the component.
#### Tests
- [ ] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

### CODE REVIEW

**Comment by levanion** [comment]
https://www.notion.so/expected/Textarea-9990c7fa79304df3abfba1ae36caabb7

**Comment by levanion** [comment]
https://www.notion.so/expected/Textarea-9990c7fa79304df3abfba1ae36caabb7

File: src/components/Textarea/Textarea.module.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+.mainWrapper {
+  position: relative;
+  width: fit-content;
+  height: fit-content;
+  display: flex;
+  flex-direction: column;
+  gap: $spacing-small;
+
+  font-style: normal;
```
Comment (jemiks) [inline src/components/Textarea/Textarea.module.scss:11]: import this styles from typography

File: src/components/Textarea/Textarea.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+          minLength={minLenght}
+          name={name}
+          onBlur={onBlur}
+          onFocus={onFocus}
+          onKeyDown={onKeyDown}
+          onChange={onEventChanged}
+          onChangeCapture={onChangeCapture}
+          onInvalid={onInvalid}
+          onInvalidCapture={onInvalidCapture}
+          onSelect={onSelect}
```
Comment (jemiks) [inline src/components/Textarea/Textarea.tsx:140]: please check the props some of these props is not relevant to the textarea,  for example, select is not the correct prop for it,
this more details here 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea

**Review by jemiks — CHANGES_REQUESTED** [review]

**Review by jemiks — CHANGES_REQUESTED** [review]

**Review by jemiks — APPROVED** [review]

**Pull request merged**